api/handlers.go

```diff
@@ -509,12 +518,19 @@ func (api *API) userLimitsSkylinkGET(u *database.User, w http.ResponseWriter, re
 		api.WriteJSON(w, respAnon)
 		return
 	}
+	// For all links that belong to MySky we return the first paid tier, so
+	// anyone can access them, even on portals which require authentication or
+	// premium accounts.
+	if _, ok := MyskyAllowlist[skylink]; ok {
```

Comment:
`nit` you can move this before the validation, not sure what `database.ValidSkylinkHash` does but it's probably just a `LoadString`? Feel free to resolve this comment if you prefer to have it here.

Reply:
I see how we don't really need validation before checking against a hardcoded list but I like having all validation done before any logic is triggered because it's nice for readability. So, I'd rather keep it here.
Thanks for your input, though!